packages/slidev/node/cli.ts

```diff
@@ -611,6 +613,14 @@ function exportOptions<T>(args: Argv<T>) {
       type: 'number',
       describe: 'scale factor for image export',
     })
+    .option('cover', {
+      type: 'boolean',
+      describe: 'prepend cover to handout, needs handout-cover.vue in project',
+    })
+    .option('handout', {
+      type: 'boolean',
+      describe: 'Export handout with slides on top and notes on bottom, optionally prepending a cover',
+    })
```

Comment:
Could be `--type=handout` (default to slides). So later, we could merge `export-notes` to `--type=notes` and unify into a single command.

Reply:
Oh I see @KermanX is doing `--template` in https://github.com/slidevjs/slidev/pull/1513

We could first discuss there